perfkitbenchmarker/configs/default_config_constants.yaml

```diff
@@ -40,11 +40,11 @@ default_single_core: &default_single_core
 
 default_500_gb: &default_500_gb
   GCP:
-    disk_type: standard
+    disk_type: pd-standard
```

Comment:
AWS is also handled by this update, and Azure is handled to the extent that we can handle it (you still need to pass the `--azure_storage_type` flag, but that doesn't change the current behavior). For the others, passing `standard` is the right thing to do right now, because those providers will translate `standard` to some appropriate provider-specific disk name. As they're updated for the refactor, we will have to also update the entries in this table.